### DISCUSSION TITLE
How to exclude Author metadata from search

### DISCUSSION THREAD

**Discussion by mhazebroucq** [microsoft-search/pnp-modern-search#2424] [Q&A] [answered]
Hi All,

I don't find how to exclude a metadata from search.
For exemple I would like to be able to search for "maxime" but I don't want to get all documents created or modified by "Maxime XXX".
Is it possible ?

I use Pnp modern search V4.

Thanks for your help

**Comment by wobba** [comment]
`NOT EditorOWSUser:"foo" NOT AuthorOWSUser:"foo"` should do it.

**Reply by mhazebroucq** [reply]
No because I don't want to exclude docs created/modified by "foo", I want docs with "foo" in content, no matter if they have been edited by foo. 
To avoid getting all documents edited by "foo" because this is not relevant for my search.

**Reply by wobba** [reply, marked as answer]
Sample with query term
`bar NOT EditorOWSUser:"john" NOT AuthorOWSUser:"john"`

this query will return all documents with "bar", but not if it's modified by "john"
